pkg/combustion/certificates.go

```diff
@@ -0,0 +1,79 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"github.com/suse-edge/edge-image-builder/pkg/template"
+	"go.uber.org/zap"
+)
+
+const (
+	certsComponentName = "certificates"
+	certsScriptName    = "30-certificates.sh"
```

Comment:
This seems rather low on the priority list. The task itself mentions:
> but likely needs to do it before Elemental/SUMA/Proxy is used.

I'm also wondering if this component should also differentiate the usages e.g. we'd eventually want to be able to provide custom certificates for Kubernetes clusters ([RKE2 docs for reference](https://docs.rke2.io/security/certificates#using-custom-ca-certificates)). Is this something we'd track separately? @rdoxenham

Reply:
I agree, this should be moved to a much higher priority in my opinion. If the proxy has a self-signed certificate (or they want to make sure it's actually the right proxy server being used), or the same situation for SUMA/Elemental/etc. then we need to make sure that the certificates are imported much earlier in the process. Suggest we go with 07 so it runs before the proxy is setup, especially as the proxy may be required for *other* things.

Reply:
@atanasdinov yes, we would definitely want to provide a way of specifying Kubernetes CA certificates. Maybe a subdirectory within the `cerfificates/` directory (or perhaps one inside of `kubernetes/`?) would be great to have at a later date. We should track this separately.

Reply:
Probably it makes more sense for those to be under `kubernetes/` then. Thanks for the clarification.